barretenberg/acir_tests/gen_inner_proof_inputs_ultra_honk.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
new one just for honk, could just merge into one potentially

Reply:
if we are going to get rid of the plonk one i could be OK without a merge

Reply:
that's true, will just delete this down the line